DSharpPlus/Metrics/LiveRestCollection.cs

```diff
@@ -0,0 +1,16 @@
+namespace DSharpPlus.Metrics;
+
+internal record struct LiveRestCollection
```

Comment:
maybe rename to ´LiveRestMetricCollection´ so you dont have to look at the namespace when seeing in the wild. Also i would use either RequestMetrics **or** RestMetrics.